pallets/tokens/src/multi_token_imbalances.rs

```diff
@@ -6,6 +6,22 @@ use frame_support::traits::{Imbalance, TryDrop};
 use sp_runtime::traits::{Saturating, Zero};
 use sp_std::{mem, result};
 
+pub trait MultiTokenImbalanceWithZeroTrait<CurrencyId> {
+    fn from_zero(currency_id: CurrencyId) -> Self;
```

Comment:
wouldnt zero be a more appropriate name ?

Reply:
zero already exists as a function in the Imbalance trait